docs/encrypted_storage.md

```diff
@@ -0,0 +1,51 @@
+# Aries Encrypted Storage
+
+The Aries framework can be figured to use encrypted storage. A concrete [EDV](https://identity.foundation/confidential-storage/) implementation is provided, but custom encrypted storage solutions can be injected.
+
+## Configuring Aries Framework with EDV storage in Go Code
+```
+// Create an edv.EncryptedFormatter with the necessary key data and crypto data to handle the conversion between unencrypted and encrypted data.
+// Check the unit tests to see an example of how you can create the jweEncrypter, jweDecrypter, and macCrypto objects.
+encryptedFormatter := edv.NewEncryptedFormatter(jweEncrypter, jweDecrypter, macCrypto)
+
+// Create an edv.RESTProvider to handle EDV server communication. It uses a single vault for all stores, and uses macCrypto for encrypted index creation.
+// The vault MUST already exist; edv.RESTProvider won't create one for you.
+edvRESTProvider, err := edv.NewRESTProvider("SomeEDVServerURL, "SomeVaultID, macCrypto)
+
+// Create the FormattedProvider, which uses encryptedFormatter and edvRESTProvider.
+formattedProvider, err := formattedstore.NewFormattedProvider(edvRESTProvider, encryptedFormatter, false)
+
+// Instantiate the framework with the formattedProvider
+framework, err := aries.New(aries.WithStoreProvider(formattedProvider))
+```
+
+## Optional Features
+
+### Caching
+
+For improved performance, you may want to enable caching by instantiating the `FormattedProvider` with the `WithCacheProvider` option.
+
+For example:
+```
+memProvider := mem.NewProvider()
+
+// Create the FormattedProvider with the WithCacheProvider option.
+formattedProvider, err := formattedstore.NewFormattedProvider(edvRESTProvider, encryptedFormatter, false, WithCacheProvider(memProvider))
+```
+
+## EDV Extensions (Optional Features)
```

Comment:
Is there an issue created in the edv spec repo for these?

Reply:
Yes

Reply:
Can you provide the links, please. @DRK3

Reply:
@troyronda Sure:

Batch: https://github.com/decentralized-identity/confidential-storage/issues/138
Returning full documents on query: https://github.com/decentralized-identity/confidential-storage/issues/137

Reply:
I also think references in the document would be helpful? @DRK3

Reply:
@troyronda Done - take a look at the updated doc and let me know if it looks ok